### ISSUE TITLE
Cause 3: Something went wrong: ENS missing on a device that had it all the time

### ISSUE THREAD

**Issue by Ein-Tim** [corona-warn-app/cwa-app-android#5911]
## Avoid duplicates
* [X] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [X] Bug is specific for Android only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [X] Bug is not already reported in another issue

## Technical details

- Device name: Huawei P Smart 2019
- Android version: 10
- App version: 3.1.1

## Describe the bug

On my Huawei P Smart 2019, which had the ENS since the very start of the COVID-19 pandemic, I suddenly can't activate exposure logging anymore. The app says that while the app is installed correctly, the "COVID-19 Exposure Notifications System" is not available on my OS. And indeed, the ENS related settings did disappear from my system settings. 

## Steps to reproduce the issue

Not sure, there was nothing special I did.

## Expected behavior

The ENS should be available.

**Comment by Ein-Tim** [comment]
@MikeMcC399 Thank you, I tried switching Bluetooth off and on, but that didn't change anything, sadly.

I don't think I will invest much more time into this issue, as the Huawei device is only a secondary device I use. I just wanted to open this issue here, as it seemed strange to me that the whole ENS just disappeared from one day to another.
This should only happen if a Google Play Service update removes it or has a bug in that regard. I will leave this issue open so that the CWA team can follow up if they want to, e.g. to ensure that Google is not removing ENS functionality on purpose already via Play Service updates.

**Comment by Ein-Tim** [comment]
@MikeMcC399 

I have version 20.15.16 (120400-309763488) of the Google Play services installed on my device. I resetted this setting once, ca. two weeks ago, but CWA appeared to be working correctly until yesterday, it was only today that I saw the error "CAUSE 3". 

I performed all updates that were offered to me via the Google Play Store and I followed all support steps described in https://support.google.com/googleplay/answer/9037938?hl=de, but the ENS settings did not re-appear. Restarting my device also didn't help

**Comment by Ein-Tim** [comment]
After some research, it appears to me that Google Play services version 20.15.16 appears to be a version from nearly 3 years ago. That would explain why the ENS is not working. I sadly can't remember the Google Play services version which was installed when I performed a reset of the ENS settings, but as I said above, this reset was nearly two weeks ago and the app worked flawlessly until yesterday. 

Does anyone know how I could update my Google Play services?

**Comment by MikeMcC399** [comment]
@Ein-Tim 

> Does anyone know how I could update my Google Play services?

To update your Google Play services (assuming that nothing else is broken on your device)

Apps > Google Play services
Scroll down to "App details in store" and tap on this
In the Google Play screen if there is an update available there will be an "Update" button showing.

`23.10.15` is a current version which I am seeing on my Samsung device.

**Comment by Ein-Tim** [comment]
> Apps > Google Play services
Scroll down to "App details in store" and tap on this

The button "App details in store" is missing for me, but I just used the direct link https://play.google.com/store/apps/details?id=com.google.android.gms&hl=en_US to access the Google Play services Google Play store page and pressed on the button "Update". After restarting the phone again, exposure logging is now working again!

I guess the reset caused the phone to use some older version of the Play services to be used again, but it's strange that CWA only stopped working now and not when I performed the reset two weeks ago. 

Anyway, this issue is solved and Google is not removing the ENS via Play services update. (relief :D)
